<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnSolicitar</class>
 <widget class="QMainWindow" name="vtnSolicitar">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solicitud</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{
	background-color: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2198c0, stop: 1 yellow);
}

QPushButton:pressed{
	background-color: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2198c0, stop: 1 green);
}
QComboBox{
	subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #86E094;
    selection-color: yellow;;
  	border-style: solid;
    border-width: 2px;
    border-radius: 10px;
	border-color: #D0C293;
	color: black;
	font: bold 12px
}
QComboBox:hover{
	border: 1px solid  #D0C293;
	color: yellow;
}
QComboBox::drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
     color: white;
     border-left-width: 0px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
     padding-left: 10px;
 }

QHeaderView::section {
    background-color: #A3E6EE;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: #D0C293;
	font: bold 14px;
	padding: 6px;
}

QTableWidget {
	background-color:  QLinearGradient(x1: 5, y1: 0, x2: 0, y2: 3, stop: 0 #2198c0, stop: 1 #F1EEE1);
    gridline-color: #E4EE9A;
     border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: #D0C293;
	font: bold 14px;
	min-width: 10em;
    padding: 6px;
}

QTableWidget QTableCornerButton::section {
    background-color: white;
     border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: #D0C293;
	padding: 6px;
}

QMainWindow{
	 background-color:  QLinearGradient(x1: 5, y1: 0, x2: 0, y2: 3, stop: 0 #2198c0, stop: 1 #F1EEE1);
}
QPushButton {
            margin: 4px;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: #D0C293;
	font: bold 14px;
	min-width: 10em;
    padding: 6px;
            background-color: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2198c0, stop: 1 #A3E6EE);
 }

QLineEdit
{
    border: 2px solid #006080;
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	border-color: #D0C293;
	font: bold 14px;
	min-width: 10em;
    padding: 6px;
}

QLineEdit:focus
{
    border: 2px solid #006080;
}

QLabel{
		font: bold 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblCedula">
            <property name="text">
             <string>Cedula</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leCedula"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lblNombre">
              <property name="text">
               <string>Nombre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leNombre"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblApellido">
              <property name="text">
               <string>Apellido</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leApellido"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblSeccion">
            <property name="text">
             <string>Seccion</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leSeccion"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lblTelefono">
            <property name="text">
             <string>Telefono</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="leTelefono"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblCota">
            <property name="text">
             <string>Cota</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leCota">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblAutor">
            <property name="text">
             <string>Autor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leAutor">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lblTItulo">
            <property name="text">
             <string>Titulo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leTitulo">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lblEjemplar">
            <property name="text">
             <string>Ejemplar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leEjemplar">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="lblCarrera">
            <property name="text">
             <string>Carrera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leCarrera">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblSemestre">
            <property name="text">
             <string>Semestre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSemestre">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="btnSolicitar">
            <property name="text">
             <string>Solicitar</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSalir">
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSalir</sender>
   <signal>clicked()</signal>
   <receiver>vtnSolicitar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
